ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-master-postsubmits.yaml

```diff
@@ -0,0 +1,56 @@
+postsubmits:
```

Comment:
Do we need a [postsubmits](https://docs.ci.openshift.org/docs/architecture/ci-operator/#post-submit-tests) file?